src/components/views/elements/ErrorBoundary.tsx

```diff
@@ -110,9 +110,9 @@ export default class ErrorBoundary extends React.PureComponent<{}, IState> {
                 <div className="mx_ErrorBoundary_body">
                     <h1>{ _t("Something went wrong!") }</h1>
                     { bugReportSection }
-                    <AccessibleButton onClick={this.onClearCacheAndReload} kind='danger'>
+                    { MatrixClientPeg.get() && <AccessibleButton onClick={this.onClearCacheAndReload} kind='danger'>
```

Comment:
It would be beneficial to add a little comment explaining the context here.
It's not obvious to me that `MatrixClientPeg.get()` returning a null-ish value means that the client was not initialised 🤔